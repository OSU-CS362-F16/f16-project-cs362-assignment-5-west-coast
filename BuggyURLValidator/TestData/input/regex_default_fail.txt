http://?df.ws/123
ftp://foo.bar/baz
http://foo.bar/?q=Test%20URL-encoded%20stuff
http://1.1.1.1.1
http://123.123.123