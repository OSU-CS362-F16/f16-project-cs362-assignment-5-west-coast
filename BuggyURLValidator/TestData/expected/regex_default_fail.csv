testcase,scheme,ALLOW_LOCAL_URLS,NO_FRAGMENTS,ALLOW_2_SLASHES,ALLOW_ALL_SCHEMES,regex,http://?df.ws/123,ftp://foo.bar/baz,http://foo.bar/?q=Test%20URL-encoded%20stuff,http://1.1.1.1.1,http://123.123.123
0,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
1,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
2,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
3,{[http][https][ftp]},OFF,OFF,OFF,ON,DEFAULT,false,false,false,false,false
4,{[http][https][ftp]},OFF,OFF,ON,OFF,DEFAULT,false,false,false,false,false
5,{[http][https][ftp]},OFF,OFF,ON,ON,DEFAULT,false,false,false,false,false
6,{[http][https][ftp]},OFF,ON,OFF,OFF,DEFAULT,false,false,false,false,false
7,{[http][https][ftp]},OFF,ON,OFF,ON,DEFAULT,false,false,false,false,false
8,{[http][https][ftp]},OFF,ON,ON,OFF,DEFAULT,false,false,false,false,false
9,{[http][https][ftp]},OFF,ON,ON,ON,DEFAULT,false,false,false,false,false
10,{[http][https][ftp]},ON,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
11,{[http][https][ftp]},ON,OFF,OFF,ON,DEFAULT,false,false,false,false,false
12,{[http][https][ftp]},ON,OFF,ON,OFF,DEFAULT,false,false,false,false,false
13,{[http][https][ftp]},ON,OFF,ON,ON,DEFAULT,false,false,false,false,false
14,{[http][https][ftp]},ON,ON,OFF,OFF,DEFAULT,false,false,false,false,false
15,{[http][https][ftp]},ON,ON,OFF,ON,DEFAULT,false,false,false,false,false
16,{[http][https][ftp]},ON,ON,ON,OFF,DEFAULT,false,false,false,false,false
17,{[http][https][ftp]},ON,ON,ON,ON,DEFAULT,false,false,false,false,false
18,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
19,{[http][https]},OFF,OFF,OFF,ON,DEFAULT,false,false,false,false,false
20,{[http][https]},OFF,OFF,ON,OFF,DEFAULT,false,false,false,false,false
21,{[http][https]},OFF,OFF,ON,ON,DEFAULT,false,false,false,false,false
22,{[http][https]},OFF,ON,OFF,OFF,DEFAULT,false,false,false,false,false
23,{[http][https]},OFF,ON,OFF,ON,DEFAULT,false,false,false,false,false
24,{[http][https]},OFF,ON,ON,OFF,DEFAULT,false,false,false,false,false
25,{[http][https]},OFF,ON,ON,ON,DEFAULT,false,false,false,false,false
26,{[http][https]},ON,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
27,{[http][https]},ON,OFF,OFF,ON,DEFAULT,false,false,false,false,false
28,{[http][https]},ON,OFF,ON,OFF,DEFAULT,false,false,false,false,false
29,{[http][https]},ON,OFF,ON,ON,DEFAULT,false,false,false,false,false
30,{[http][https]},ON,ON,OFF,OFF,DEFAULT,false,false,false,false,false
31,{[http][https]},ON,ON,OFF,ON,DEFAULT,false,false,false,false,false
32,{[http][https]},ON,ON,ON,OFF,DEFAULT,false,false,false,false,false
33,{[http][https]},ON,ON,ON,ON,DEFAULT,false,false,false,false,false
34,{[http][https][ftp]},OFF,OFF,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true
35,{[http][https][ftp]},OFF,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
36,{[http][https][ftp]},OFF,OFF,ON,OFF,^[\w-\\.]*$,true,true,true,true,true
37,{[http][https][ftp]},OFF,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true
38,{[http][https][ftp]},OFF,ON,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true
39,{[http][https][ftp]},OFF,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
40,{[http][https][ftp]},OFF,ON,ON,OFF,^[\w-\\.]*$,true,true,true,true,true
41,{[http][https][ftp]},OFF,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true
42,{[http][https][ftp]},ON,OFF,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true
43,{[http][https][ftp]},ON,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
44,{[http][https][ftp]},ON,OFF,ON,OFF,^[\w-\\.]*$,true,true,true,true,true
45,{[http][https][ftp]},ON,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true
46,{[http][https][ftp]},ON,ON,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true
47,{[http][https][ftp]},ON,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
48,{[http][https][ftp]},ON,ON,ON,OFF,^[\w-\\.]*$,true,true,true,true,true
49,{[http][https][ftp]},ON,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true
50,{[http][https]},OFF,OFF,OFF,OFF,^[\w-\\.]*$,true,false,true,true,true
51,{[http][https]},OFF,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
52,{[http][https]},OFF,OFF,ON,OFF,^[\w-\\.]*$,true,false,true,true,true
53,{[http][https]},OFF,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true
54,{[http][https]},OFF,ON,OFF,OFF,^[\w-\\.]*$,true,false,true,true,true
55,{[http][https]},OFF,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
56,{[http][https]},OFF,ON,ON,OFF,^[\w-\\.]*$,true,false,true,true,true
57,{[http][https]},OFF,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true
58,{[http][https]},ON,OFF,OFF,OFF,^[\w-\\.]*$,true,false,true,true,true
59,{[http][https]},ON,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
60,{[http][https]},ON,OFF,ON,OFF,^[\w-\\.]*$,true,false,true,true,true
61,{[http][https]},ON,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true
62,{[http][https]},ON,ON,OFF,OFF,^[\w-\\.]*$,true,false,true,true,true
63,{[http][https]},ON,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true
64,{[http][https]},ON,ON,ON,OFF,^[\w-\\.]*$,true,false,true,true,true
65,{[http][https]},ON,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true
66,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
